CHANGES.rst

```diff
@@ -228,6 +228,14 @@ API Changes
   - Moved ``units.cgs.emu`` to ``units.deprecated.emu`` due to ambiguous
     definition of "emu". [#4918, #5906]
 
+  - ``jupiterMass``, ``earthMass``, ``jupiterRad``, and ``earthRad`` no longer
+    have their prefixed units included in the standard units.  If needed, they
+    can still  be found in ``units.deprecated``. [#5661]
+    
+  - ``solLum``,``solMass``, and ``solRad`` no longer have  their prefixed units
+    included in the standard units.  If needed, they can still be found in  
+    ``units.vounit_needed``, and are enabled by default. [#5661]
```

Comment:
Hmm, I see that.  It's really `required_by_vounit_buy_not_in_units`, but that's still awkwardly long.